### PR TITLE
faq: update TiDB Lightning FAQ

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 

- Update three FAQs about TiDB Lighting
### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1837

### Which version does your change affect? 
dev, v3.0, v2.1

### CODE REVIEW

File: v2.1/faq/tidb-lightning.md [hunk 1]
```diff
@@ -79,27 +79,29 @@ Yes, as long as every `tidb-lightning` instance operates on different tables.
 
 ## How to stop `tikv-importer`?
 
-If it is deployed using TiDB Ansible, run `scripts/stop_importer.sh` under the deployed folder.
+You can choose the corresponding operation to stop `tikv-importer` according to your deployment method.
 
-Otherwise, obtain the process ID with `ps aux | grep tikv-importer`, and then run `kill «pid»`.
+- For deployment using TiDB Ansible: run `scripts/stop_importer.sh` on the Importer server.
+
+- For manual deployment: if `tikv-importer` is running in foreground, press <kbd>Ctrl</kbd>+<kbd>C</kbd> to exit. Otherwise, obtain the process ID using the `ps aux | grep tikv-importer` command and then end the process using the `kill «pid»` command.
```
Comment (lance6716) [inline v2.1/faq/tidb-lightning.md]: ditto

File: v3.0/faq/tidb-lightning.md [hunk 2]
```diff
@@ -80,27 +80,29 @@ Yes, as long as every `tidb-lightning` instance operates on different tables.
...
 ## How to stop `tidb-lightning`?
 
-If it is deployed using TiDB Ansible, run `scripts/stop_lightning.sh` under the deployed folder.
+You can choose the corresponding operation to stop `tidb-lightning` according to your deployment method.
 
-If `tidb-lightning` is running in foreground, simply press <kbd>Ctrl</kbd>+<kbd>C</kbd> to stop it.
+- For deployment using TiDB Ansible: run `scripts/stop_lightning.sh` on the Lightning server.
 
-Otherwise, obtain the process ID with `ps aux | grep tidb-importer`, then run `kill -2 «pid»`.
+- For manual deployment: if `tidb-lightning` is running in foreground, press <kbd>Ctrl</kbd>+<kbd>C</kbd> to exit. Otherwise, obtain the process ID using the `ps aux | grep tidb-lighting` command and then end the process using the `kill -2 «pid»` command.
```
Comment (lance6716) [inline v3.0/faq/tidb-lightning.md]: ditto

**Comment by lance6716** [comment]
LGTM

**Comment by juliezhang1112** [comment]
PTAL @yikeke

File: dev/faq/tidb-lightning.md [hunk 3]
```diff
@@ -103,27 +103,29 @@ Yes, as long as every `tidb-lightning` instance operates on different tables.
 
 ## How to stop `tikv-importer`?
```
Comment (yikeke) [inline dev/faq/tidb-lightning.md]:
```suggestion
## How to stop the `tikv-importer` process?
```

File: dev/faq/tidb-lightning.md [hunk 4]
```diff
@@ -103,27 +103,29 @@ Yes, as long as every `tidb-lightning` instance operates on different tables.
...
 
-If it is deployed using TiDB Ansible, run `scripts/stop_importer.sh` under the deployed folder.
+You can choose the corresponding operation to stop `tikv-importer` according to your deployment method.
 
-Otherwise, obtain the process ID with `ps aux | grep tikv-importer`, and then run `kill «pid»`.
+- For deployment using TiDB Ansible: run `scripts/stop_importer.sh` on the Importer server.
+
+- For manual deployment: if `tikv-importer` is running in foreground, press <kbd>Ctrl</kbd>+<kbd>C</kbd> to exit. Otherwise, obtain the process ID using the `ps aux | grep tikv-importer` command and then terminate the process using the `kill «pid»` command.
 
 ## How to stop `tidb-lightning`?
```
Comment (yikeke) [inline dev/faq/tidb-lightning.md]:
```suggestion
## How to stop the `tidb-lightning` process?
```

File: dev/faq/tidb-lightning.md [hunk 5]
```diff
@@ -103,27 +103,29 @@ Yes, as long as every `tidb-lightning` instance operates on different tables.
...
-Otherwise, obtain the process ID with `ps aux | grep tidb-importer`, then run `kill -2 «pid»`.
+- For manual deployment: if `tidb-lightning` is running in foreground, press <kbd>Ctrl</kbd>+<kbd>C</kbd> to exit. Otherwise, obtain the process ID using the `ps aux | grep tidb-lighting` command and then terminate the process using the `kill -2 «pid»` command.
 
 ## Why `tidb-lightning` suddenly quits while running in background?
 
-It is potentially caused by starting `tidb-lightning` incorrectly, which causes the system to send a SIGHUP signal to stop it. If this is the case, there should be a log entry like:
+It is potentially caused by starting `tidb-lightning` incorrectly, which causes the system to send a SIGHUP signal to stop `tidb-lightning`. In this situation, `tidb-lightning.log` usually outputs the following log:
 
 ```
 2018/08/10 07:29:08.310 main.go:47: [info] Got signal hangup to exit.
```
Comment (yikeke) [inline dev/faq/tidb-lightning.md]:
```suggestion
[2018/08/10 07:29:08.310 +08:00] [INFO] [main.go:41] ["got signal to exit"] [signal=hangup]
```

**Comment by juliezhang1112** [comment]
PTAL @yikeke

**Pull request merged**